libs-scala/grpc-reverse-proxy/BUILD.bazel

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//bazel_tools:scala.bzl",
+    "da_scala_library",
+    "da_scala_test_suite",
+)
+
+da_scala_library(
```

Comment:
Needs more Scala 2.13 :slightly_smiling_face:

Reply:
Does it? It passes tests with both versions, doesn't it? 🤔

Reply:
oh good point duh, I got confused by the lack of `scala_deps` but you just don’t have Scala dependencies.

Reply:
As a side note, the gRPC Java API makes a lot of sense for Java, but probably there are things that can be improved to make it simpler and more typesafe to use via Scala. Not sure though, just a general sense I got while using it.